tests/dualtor/conftest.py

```diff
@@ -2,9 +2,13 @@
 import logging
 import time
 
-from tests.common.dualtor.dual_tor_utils import get_crm_nexthop_counter # lgtm[py/unused-import]
+pytest_plugins = (
+    'tests.common.dualtor.dual_tor_utils',
+    'tests.common.dualtor.dual_tor_mock'
+)
```

Comment:
These are very extensive imports. Do we need all these plugins?

Reply:
actually I see most of fixtures within mentioned files are used by dualtor tests, directly or indirectly (as intermediate fixture dependencies). So I think we need to import them all at once globally for dualtor tests, so specific tests won't need to be aware (do import) of the fixtures they don't directly depend on